src/Microsoft.AspNetCore.Blazor.Browser/Hosting/BrowserHostingEnvironment.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Blazor.Hosting
+{
+    internal class BrowserHostingEnvironment : IBrowserHostingEnvironment
+    {
+    }
```

Comment:
So this is one of the features of the web/generic host that we might want to parallel if we have any useful data to plug in here. Thoughts? If we don't have ideas I'll take it out.

Reply:
If our only scenario was browser hosting, then I think this would be pretty much definitely something we'd want eventually (e.g., to indicate development vs production environment).

But since we will have other scenarios too, I'm unsure what would end up on `IBrowserHostingEnvironment` as opposed to something more general and not browser specific.

Personally I'd err on the side of leaving this out until the use case becomes clearer, but if you feel there are some clear cases where we'd want a browser-specific hosting environment concept then it's fine with me.

Reply:
OK I'll take this out for now.